code/__DEFINES/preferences.dm

```diff
@@ -65,8 +65,9 @@
 #define TOGGLES_DEADCHAT_DEFAULT (NONE)
 
 #define RADIAL_MEDICAL (1<<0)
+#define MIDDLESHIFTCLICKING (1<<1)
```

Comment:
I would appreciate if these were both tabbed out to some common level. Just so long as they are both on the same `(1<<0)` level, not unlike other bitshift defines.

Reply:
tabbed out? I don't understand what you are asking

Reply:
Dont